### PR TITLE
Document signing of Git Commits

### PR DESCRIPTION
**Pull request by juliogonzalez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,12 +80,14 @@ The script `clean-old-results` can be used to get rid of undesired old results (
 
 ## How to contribute
 
-It is easy: just create a Pull Request with your contribution, and make sure that the unit tests are passing.
+It is easy: 
+- Make sure you have Git commit signing enabled. If you are not doing it already, check out the [GitHub documentation](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification).
```
Comment (nodeg) [inline README.md]:
```suggestion
It is easy:

- Make sure you have Git commit signing enabled. If you are not doing it already, check out the [GitHub documentation](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification).
```
Trailing whitespace and lists should be surrounded by blank lines.

File: README.md [hunk 2]
```diff
@@ -80,12 +80,14 @@ The script `clean-old-results` can be used to get rid of undesired old results (
...
+- Make sure unit tests are passing
+  To run them:
+  ```bash
+  python3 -m unittest
+  ```
+  For now the unit tests only cover the module.
+- Then just Pull Request with your contribution.
```
Comment (nodeg) [inline README.md]: There are also too many newlines at the end, if I have seen this correctly in my local editor.

**Review by nodeg — COMMENTED** [review]
Small nitpicks.

File: README.md [hunk 3]
```diff
@@ -80,12 +80,14 @@ The script `clean-old-results` can be used to get rid of undesired old results (
 
 ## How to contribute
 
-It is easy: just create a Pull Request with your contribution, and make sure that the unit tests are passing.
+It is easy: 
+- Make sure you have Git commit signing enabled. If you are not doing it already, check out the [GitHub documentation](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification).
+- Make sure unit tests are passing
```
Comment (nodeg) [inline README.md]:
```suggestion
- Make sure unit tests are passing.
```
The rendered version does not automatically add a new line as one would assume when looking at your changes.
image

File: README.md [hunk 4]
```diff
@@ -80,12 +80,14 @@ The script `clean-old-results` can be used to get rid of undesired old results (
...
+  To run them:
+  ```bash
+  python3 -m unittest
+  ```
+  For now the unit tests only cover the module.
+- Then just Pull Request with your contribution.
```
Comment (nodeg) [inline README.md]:
```suggestion
- Then, just open a Pull Request with your contribution.
```

File: README.md (hunk 1 above)
Comment (juliogonzalez) [reply README.md]: Why? Works fine without it:

imagen

File: README.md (hunk 3 above)
Comment (juliogonzalez) [reply README.md]: Ah. This one one yes. Fixing it.

File: README.md (hunk 1 above)
Comment (nodeg) [reply README.md]: https://github.com/DavidAnson/markdownlint/blob/v0.34.0/doc/md032.md :wink:

**Comment by nodeg** [comment]
With your last commit, you broke the list layout:

image

compared to

image

File: README.md (hunk 1 above)
Comment (juliogonzalez) [reply README.md]: *sigh*, so it's about some parses not working correctly:
> Rationale: Aside from aesthetic reasons, some parsers, including kramdown, will not parse lists that don't have blank lines before and after them.

The aesthetic reasons could be discussed here, as the paragraph is the explanation for the list, but OK.

**Comment by juliogonzalez** [comment]
imagen

**Review by nodeg — APPROVED** [review]
The list layout is still "broken", but it's fine. We have more Markdown issues in the README.
Thank you, Julio.

**Comment by juliogonzalez** [comment]
> The list layout is still "broken", but it's fine. We have more Markdown issues in the README. Thank you, Julio.

What's broken? I see it correctly on the GitHub preview.

**Comment by juliogonzalez** [comment]
BTW @nodeg, if we really want to enforce such rules, there should be actions telling the users about that (at every repo where there're markdown files :-))

In the end we are talking about linting, not about syntax users, and most contributors (myself included), are not really aware of all those rules, and won't remember about them for casual contributions.

**Comment by nodeg** [comment]
Makes sense. An idea would be to have a GitHub Actions for linting Markdown files when opening a Pull Request.

**Pull request merged**